test/config-generator.js

```diff
@@ -466,6 +465,33 @@ describe('The config-generator function', () => {
             const actualConfig = configGenerator(config);
             expect(actualConfig.devServer).to.be.undefined;
         });
+
+        it('devServer no hot mode', () => {
+            const config = createConfig();
+            config.runtimeConfig.useDevServer = true;
+            config.runtimeConfig.devServerUrl = 'http://localhost:8080/';
+            config.runtimeConfig.useHotModuleReplacement = false;
+            config.outputPath = '/tmp/public';
```

Comment:
it doesn't matter here, because it's not what I'm testing for. I just need a path that won't cause the code to throw an exception internally (`Unable to determine contentBase option...`) - and that logic for that exception *is* tested elsewhere with the proper paths. We could probably clean this file up a bit with some mocks.